mne/viz/utils.py

```diff
@@ -2765,6 +2764,8 @@ def center_cmap(cmap, vmin, vmax, name="cmap_centered"):
     here:
     https://matplotlib.org/users/colormapnorms.html#custom-normalization-two-linear-ranges
     """  # noqa: E501
+    from matplotlib.colors import LinearSegmentedColormap
```

Comment:
Hooray automated Travis pedantry! :)

Reply:
Yep, but also boo stupid PyCharm regression: https://youtrack.jetbrains.com/issue/PY-32565
:smile:

Reply:
Shhh... don't mention IDEs within earshot of @agramfort or he might disown you